packages/core/test/controls/radioGroupTests.tsx

```diff
@@ -100,6 +100,17 @@ describe("<RadioGroup>", () => {
         assert.lengthOf(group.find(Radio), 2);
     });
 
+    it("passes custom classNames to options", () => {
+        const OPTIONS = [{ className: "apple", value: "a" }, { className: "banana", value: "b" }];
+        const group = mount(<RadioGroup onChange={emptyHandler} options={OPTIONS} selectedValue="a" />);
+        assert.strictEqual(findRadio(group, { value: "a" }).prop("className"), "apple");
+        assert.strictEqual(findRadio(group, { value: "b" }).prop("className"), "banana");
```

Comment:
enzyme provides `wrapper.hasClass(name): boolean`, but this is ok as error messages will be different.

Reply:
but do move this near the other `options` tests